skimage/transform/tests/test_warps.py

```diff
@@ -436,6 +436,23 @@ def test_downscale_anti_aliasing():
     assert_equal(scaled[3:, :].sum(), 0)
     assert_equal(scaled[:, 3:].sum(), 0)
 
+    sigma = 0.12499999  # The filter will be three pixels wide
```

Comment:
After reading only the edits, I'm not sure to understand how sigma=0.13 gives 3 pixels wide... and a future reader will probably have the same question.

Reply:
The Gaussian filter is truncated after 4 standard deviations, so the filter width will be 0.125 * 4 = 1, and the filter is constructed as np.arange(-width, witdth + 1). Hence, for width = 1, np.arange(-1, 2) = np.array([-1, 0, 1]). This particular constant is from some tests I ran, and is no longer important.

I've therefore updated it so that sigma = 0.125, and have removed the comment since the filter width is of no matter.

I've also moved the tests from test_downscale_anti_aliasing to test_downsize_anti_aliasing, because that is the right place for them.